tests/src/integration/zipaction/actions/cat/index.js

```diff
@@ -0,0 +1,5 @@
+function myAction(args) {
+    var cat = require('cat');
+    cat('https://baidu.com', console.log); 
```

Comment:
Change this to a neutral website (like openwhisk.org) <or even better> do not require HTTPS and "cat" some local file?